components/rpill/browser/rpill.h

```diff
@@ -0,0 +1,15 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_RPILL_BROWSER_RPILL_H_
+#define BRAVE_COMPONENTS_RPILL_BROWSER_RPILL_H_
+
+#include <string>
+
+namespace rpill {
+bool exec();
```

Comment:
can we have a more meaningful name per my previous comment on the rust code?

Reply:
Changed to `bool validate_os_win()`